app/views/admin/admin_portals/index.html.erb

```diff
@@ -1,3 +1,15 @@
+<div>
+  <div class="tag-card crayons-card p-4 m:p-6 m:pt-4 mb-4 flex flex-row relative">
+    <div class="mr-8">
+      <div class="crayons-field__description">Last deployed</div>
+      <div><%= ApplicationConfig["FOREM_BUILD_DATE"].presence || ENV["HEROKU_RELEASE_CREATED_AT"] %></div>
```

Comment:
I'm not saying we need this now, BUT I think it could make sense to add a new model (not DB backed) like `ForemInstance` to which we add a bunch of class methods that encapsulate things like this:

```ruby
class ForemInstance
  def self.deployed_at
    ApplicationConfig["FOREM_BUILD_DATE"].presence || ENV["HEROKU_RELEASE_CREATED_AT"
  end

  def self.latest_commit_id
    ApplicationConfig["FOREM_BUILD_SHA"].presence || ENV["HEROKU_SLUG_COMMIT"]
  end
end
```

We probably will have a few more and I'd maybe even move `dev_to?` from `SiteConfig` to that new class since `ForemInstance.dev_to?` makes a lot more sense to me than `SiteConfig.dev_to?`. We could even move the `local?` method for the same reason (`ForemInstance.local?`). 

I found that in a lot of Rails apps the main concept itself often has no representation in code (e.g. blogs have `Author`, `Post` and `Comment` models, but nothing to represent the whole blog), but they often come in handy to collect certain utility functions in a sensible location.

Reply:
@citizen428 this makes sense. I didn't include the changes in this PR, but I'm creating a follow up PR tomorrow. Thanks for the feedback :)

Reply:
No rush, like I said it really doesn't need to be part of this PR, but I feel like this will be a useful addition. Thanks for doing a follow-up PR! 😃